sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -124,6 +135,46 @@ instances:
     #
     # command_timeout: 5
 
+    ## @param use_global_custom_queries - string - optional - default: true
+    ## How `global_custom_queries` should be used for this instance. There are 3 options:
+    ##
+    ## 1. true - `global_custom_queries` override `custom_queries`.
+    ## 2. false - `custom_queries` override `global_custom_queries`.
+    ## 3. extend - `global_custom_queries` are used in addition to any `custom_queries`.
+    #
+    # use_global_custom_queries: 'true'
+
+    ## @param custom_queries - list of mappings - optional
+    ## Each query must have 2 fields, and can have a third optional field:
+    ##
+    ## 1. query - The SQL to execute. It can be a simple statement or a multi-line script.
+    ##            Use the pipe `|` if you require a multi-line script.
+    ## 2. columns - The list representing each column, ordered sequentially from left to right.
+    ##              The number of columns must equal the number of columns returned in the query.
+    ##              There are 2 required pieces of data:
+    ##                a. name - The suffix to append to `<INTEGRATION>.` to form
+    ##                          the full metric name. If `type` is `tag`, this column is
+    ##                          considered a tag and applied to every
+    ##                          metric collected by this particular query.
+    ##                b. type - The submission method (gauge, monotonic_count, etc.).
```

Comment:
Rather than saying "etc." can we either list all or say where the complete list can be found?

Reply:
Thanks for the review!  These changes are part of our configuration templates though, and I think @ofek has some plans to update them in the future.

Reply:
Hi everyone, we have been trying to implement a `count` type metric via stored procedure, but this isn't supported.

Is `count` supported for "custom queries"? We might be able to use this instead if so...